sources/packages/web/src/components/aest/institution/ApplicationOfferingChangeDetailsHeader.vue

```diff
@@ -0,0 +1,179 @@
+<template>
+  <div class="m-3">
+    <v-row>
+      <header-title-value
+        title="Submitted date"
+        :value="
+          headerDetails.submittedDate
+            ? dateOnlyLongString(headerDetails.submittedDate)
+            : '-'
+        "
+      />
+      <Separator v-if="headerDetails.institutionName"></Separator>
+      <header-title-value title="Institution"
+        ><template #value
+          ><span class="link-primary" @click="navigateToInstitutionProfile()">
+            {{ headerDetails.institutionName }}
+          </span>
+        </template></header-title-value
+      >
+      <Separator v-if="headerDetails.locationName" />
+      <header-title-value
+        v-if="headerDetails.locationName"
+        title="Location"
+        :value="headerDetails.locationName"
+      />
+    </v-row>
+    <v-row class="mt-4" v-if="showReviewDetails">
+      <header-title-value
+        :iconLabel="iconLabel"
+        :iconColor="iconColor"
+        :title="assessedByLabel"
+        :value="assessedByLabelValue"
+      />
+      <Separator />
+      <header-title-value
+        :title="assessedDateLabel"
+        :value="assessedDateLabelValue"
+      />
+    </v-row>
+  </div>
+</template>
+
+<script lang="ts">
+import {
+  ApplicationOfferingChangeRequestHeader,
+  ApplicationOfferingChangeRequestStatus,
+} from "@/types";
+import { useFormatters } from "@/composables";
+import { computed, defineComponent, watchEffect, ref } from "vue";
+import { useRouter } from "vue-router";
+import { AESTRoutesConst } from "@/constants/routes/RouteConstants";
+
+interface ReviewLabels {
+  assessedByLabel: string;
+  assessedDateLabel: string;
+}
+
+export default defineComponent({
+  props: {
+    headerDetails: {
+      type: Object,
+      required: true,
+      default: {} as ApplicationOfferingChangeRequestHeader,
+    },
+  },
+  setup(props) {
+    const iconLabel = ref("");
+    const iconColor = ref("");
+    const assessedByLabel = ref("");
+    const assessedByLabelValue = ref("");
+    const assessedDateLabel = ref("");
+    const assessedDateLabelValue = ref("");
+    const router = useRouter();
+    const { dateOnlyLongString } = useFormatters();
+    const showReviewDetails = computed(
+      () =>
+        (props.headerDetails.assessedBy || props.headerDetails.updatedDate) &&
+        [
+          ApplicationOfferingChangeRequestStatus.Approved,
+          ApplicationOfferingChangeRequestStatus.DeclinedBySABC,
+          ApplicationOfferingChangeRequestStatus.DeclinedByStudent,
+        ].includes(props.headerDetails.status),
+    );
+    const checkHeaderDetailsStatus = (
+      statuses: ApplicationOfferingChangeRequestStatus[],
+    ): boolean => {
+      return statuses.includes(props.headerDetails.status);
+    };
+    watchEffect(() => {
+      if (
```

Comment:
Just a possible suggestion for the next time  (IMO) this entire logic would be better expressed in a switch where it would be better to know what was expected for each status.